adapter-jdbc/src/main/resources/activities.baselinesv2/cockroachdb-keyvalue.yaml

```diff
@@ -0,0 +1,58 @@
+# run driver=jdbc workload="/path/to/cockroachdb-keyvalue.yaml" tags="block:schema" threads=AUTO cycles=4 url="jdbc:postgresql://crdb0-3153.g8z.cockroachlabs.cloud:26257/defaultdb?sslmode=verify-ca&password=CHANGE_ME&user=newuser" databaseName="defaultdb" portNumber=26257 user="newuser" password="CHANGE_ME" sslmode="verify-full" serverName=insectdb sslrootcert="/path/to/postgresql_certs/root.crt" -vv --show-stacktraces
+min_version: "5.17.1"
+
+description: |
+  A workload with only text keys and text values. This is based on the CQL keyvalue workloads as found
+  in cql-keyvalue2.yaml.
+
+scenarios:
+  default:
+    schema: run driver=jdbc workload="/path/to/cockroachdb-keyvalue.yaml" tags==block:schema   threads=1    cycles==UNDEF                             url="jdbc:postgresql://crdb0-3153.g8z.cockroachlabs.cloud:26257/defaultdb?sslmode=verify-ca&password=CHANGE_ME&user=newuser" databaseName="defaultdb" portNumber=26257 user="newuser" password="CHANGE_ME" sslmode="verify-full" serverName="crdb" sslrootcert="/path/to/postgresql_certs/root.crt"
```

Comment:
The endpoint example should probably be generalized more, such as `jdbc:postgresql://ENDPOINT:PORT/...`

Reply:
This is now addressed. 👍🏼